test/sql/copy/parquet/parquet_glob.test

```diff
@@ -69,13 +69,13 @@ FROM parquet_scan('data/parquet-testing/glob3/*/dir/*.parquet');
 statement error
 select count(*) from parquet_scan('')
 ----
-<REGEX>:.*IO Error.*can only be set for.*
+
```

Comment:
What error is thrown now? I understand that the error has changed, so you must edit this test, but we try not to use `statement error` with an empty expected result, if possible.